src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx

```diff
@@ -128,25 +131,49 @@ export const BreachDetailsView = (props: Props) => {
         </h2>
         <div className={styles.content}>
           <p>
-            {l10n.getString("breach-overview-new", {
-              breachDate: (breach.BreachDate as unknown as Date).toLocaleString(
-                getLocale(l10n),
-                {
+            {breach.Name === "BVD" && getLocale(l10n).split("0")[0] === "en" ? (
+              <>
+                In approximately August 2021,{" "}
+                <a
+                  href="https://kaduu.io/blog/2022/02/04/us-strategic-company-bureau-van-dijk-hacked/"
```

Comment:
This seemed to redirect for me and takes me to a "This article no longer exist. Please visit our news section." page.

Reply:
I'll ask Tony about that. I'd swear I saw it this morning, but that's probably not true.

Reply:
Removed in 2cda9b46f.